pkg/app/piped/controller/scheduler.go

```diff
@@ -612,6 +612,20 @@ func (s *scheduler) reportDeploymentCompleted(ctx context.Context, status model.
 
 	defer func() {
 		switch status {
+
+		case model.DeploymentStatus_DEPLOYMENT_ROLLING_BACK:
```

Comment:
`status` cannot be `DeploymentStatus_DEPLOYMENT_ROLLING_BACK` in the current implementation. See [here](https://github.com/pipe-cd/pipecd/blob/46fd914f2852da0a3d9d5c7ee711402207b3c621/pkg/app/piped/controller/scheduler.go#L404)

Reply:
I wrote my opinion [here](https://github.com/pipe-cd/pipecd/issues/4547#issuecomment-1694820677).
Thank you @sivchari !